po/zh_CN.po

```diff
@@ -701,6 +701,76 @@ msgstr ""
 msgid "Whether watchdog integration is enabled"
 msgstr "是否启用watchdog集成设置"
 
+#: lib/pengine/common.c:120
+msgid ""
+"This is set automatically by the cluster according to whether SBD is "
+"detected to be in use. User-configured values are ignored. The value `true` "
+"is meaningful if diskless SBD is used and `stonith-watchdog-timeout` is "
+"nonzero. In that case, if fencing is required, watchdog-based self-fencing "
+"will be performed via SBD without requiring a fencing resource explicitly "
+"configured."
+msgstr ""
+"这是由集群根据是否 SBD 自动设置的检测到正在使用。用户配置的值将被忽略。如果使"
+"用无盘 SBD 并且值 `true` 是有意义的`stonith-watchdog-timeout` 不为零。在这种"
+"情况下，如果fencing 是必需的，基于看门狗的自我防护将通过”执行SBD 无需明确配置"
+"防护资源。"
+
+#: lib/pengine/common.c:130
+msgid "Allow performing fencing operations in parallel"
+msgstr "允许并行执行 fencing 操作"
+
+#: lib/pengine/common.c:136
+msgid "*** Advanced Use Only *** Whether to fence unseen nodes at start-up"
+msgstr "*** 仅高级使用 *** 是否在启动时屏蔽不可见节点"
+
+#: lib/pengine/common.c:137
+msgid ""
+"Setting this to false may lead to a \"split-brain\" situation,potentially "
+"leading to data loss and/or service unavailability."
+msgstr ""
+"将此设置为 false 可能会导致“脑裂”的情况，可能导致数据丢失和/或服务不可用。"
+
+#: lib/pengine/common.c:143
+msgid ""
+"Apply fencing delay targeting the lost nodes with the highest total resource "
+"priority"
+msgstr "针对具有最高总资源优先级的丢失节点应用fencing延迟"
+
+#: lib/pengine/common.c:144
+msgid ""
+"Apply specified delay for the fencings that are targeting the lost nodes "
+"with the highest total resource priority in case we don't have the majority "
+"of the nodes in our cluster partition, so that the more significant nodes "
+"potentially win any fencing match, which is especially meaningful under "
+"split-brain of 2-node cluster. A promoted resource instance takes the base "
+"priority + 1 on calculation if the base priority is not 0. Any static/random "
+"delays that are introduced by `pcmk_delay_base/max` configured for the "
+"corresponding fencing resources will be added to this delay. This delay "
+"should be significantly greater than, safely twice, the maximum "
+"`pcmk_delay_base/max`. By default, priority fencing delay is disabled."
+msgstr ""
+"对于针对丢失者的fencings应用指定的延迟如果我们不这样做，则具有最高总资源优先"
+"级的节点在我们的集群分区中拥有大多数节点，因此更重要的节点有可能赢得任何"
+"fencing比赛，这在 2 节点的脑裂下特别有意义集群。提升的资源实例采用基本优先级 "
+"+ 1如果基本优先级不为 0，则计算。任何静态/随机由 `pcmk_delay_base/max` 配置引"
```

Comment:
So far the term "promotable" is translated to "可升级",  and "promoted" to "主":

https://github.com/ClusterLabs/pacemaker/pull/2909/files#diff-0a4a9ca503c41c5f2a54de36293d8863d6ef72d42476226fa0b9eddf52bc94e9R647-R648

Let's make them consistent one way or another.

So if to comply with the other, it could be "如果基本优先级不为 0, 在计算时主资源实例获得基本优先级 +1。"